components/places/ios/Places/HistoryMetadata.swift

```diff
@@ -10,6 +10,15 @@ import Foundation
 public enum DocumentType: Int32 {
     case regular = 0
     case media = 1
+    case unknown = -1
+    
+    public init(rawValue: Int32) {
```

Comment:
So what's your use case for this? Can't you just do `DocumentType.media` when creating instances? I suppose this may be helpful when deserializing, but I don't think you'd need to persist a `DocumentType` yourself manually.

Also, these numbers aren't entirely random - they need to align w/ what other platforms are using. And currently we don't have a `-1` present elsewhere. So unless you absolutely need this change, I'd remove at least the `unknown` case.